jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-api/src/main/java/org/jbpm/workbench/cm/model/CaseRoleAssignmentSummary.java

```diff
@@ -64,6 +64,10 @@ public void setUsers(final List<String> users) {
         this.users = ofNullable(users).orElse(new ArrayList<>());
     }
 
+    public boolean hasAssignment(){
+        return getUsers().size() +getGroups().size() > 0;
+    }
+
```

Comment:
Since the method is moved here, maybe we don't need to fetch users and groups through getters but rather use them directly.